contrib/sip_proxy/filters/network/test/mocks.cc

```diff
@@ -32,7 +32,6 @@ MockDecoderCallbacks::MockDecoderCallbacks() {
   service1.set_domain("pcsf-cfed.cncs.svc.cluster.local");
   local_services_.emplace_back(service1);
   local_services_.emplace_back(service2); */
-  ON_CALL(*this, getLocalIp()).WillByDefault(Return("127.0.0.1"));
```

Comment:
Where are the test for this?

Reply:
Hi @htuch , Thanks for your comments. as a contrib extension. there is no UT coverage required. so we haven't add enouth UT codes before. But there will be a PR soon to add all our UT codes.